test/src/main/java/org/apache/accumulo/test/functional/BulkNewMetadataSkipIT.java

```diff
@@ -109,7 +109,7 @@ public void setupCluster() throws Exception {
   }
 
   @ParameterizedTest
-  @ValueSource(ints = {0, 0, 2, 4, 8, 16, 32, 64, 128})
+  @ValueSource(ints = {0, 1, 2, 4, 8, 16, 32, 64, 128})
```

Comment:
Just a heads up that the reason for the two zero values was because there might have been an initial startup cost in the first test run. If you are looking at timing information then the first run could be misleading. No issue here changing it to a `1`, just wanted to provide some background.